src/guides/v2.3/ui_comp_guide/components/ui-columns-editing-client.md

```diff
@@ -0,0 +1,71 @@
+---
+group: ui-components-guide
+title: ColumnsEditingClient component
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The ColumnsEditingClient [UI component](https://glossary.magento.com/ui-component) is an [extension](https://glossary.magento.com/extension) for the [ColumnsEditor]({{ site.mage2bloburl }}/{{page.guide_version}}/app/code/Magento/Ui/view/base/web/js/grid/editing/editor.js) component, it provides functionality for saving edited data.
+
+## Configuration options
+
+| Option | Description | Type | Default Value |
+| --- | --- | --- | --- |
+| `component` | The path to the component’s `.js` file in terms of RequireJS. | String | `Magento_Ui/js/grid/editing/client` |
```

Comment:
"in terms of RequireJS" is not clear. It should just be a relative path, correct?

Reply:
RequireJS has its own [path processing](https://requirejs.org/docs/api.html#config-paths). It's not exactly a relative path since it does not contain the file extension ('*.js'). The "RequireJS path" definition also covers a case when some JS file has an alias registered in the RequireJS `map` [configuration](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/requirejs.html#requirejs-config-map).

Reply:
Thanks for clarifying, @rogyar.  "The path to the component's `.js` file, relative to RequireJS."